pkgs/development/libraries/audio/libbass/default.nix

```diff
@@ -2,20 +2,22 @@
 
 # Upstream changes files in-place, to update:
 # 1. Check latest version at http://www.un4seen.com/
-# 2. Update `version`s and `sha256` sums.
+# 2. Update `version`s and `hash` sums.
 # See also http://www.un4seen.com/forum/?topic=18614.0
 
 let
   allBass = {
     bass = {
       h = "bass.h";
-      version = "2.4.15";
+      version = "2.4.17";
       so = {
-        i686_linux = "libbass.so";
-        x86_64-linux = "x64/libbass.so";
+        i686_linux = "libs/x86/libbass.so";
+        x86_64-linux = "libs/x86_64/libbass.so";
+        armv7l-linux = "libs/armhf/libbass.so";
+        aarch64-linux = "libs/aarch64/libbass.so";
       };
       urlpath = "bass24-linux.zip";
-      sha256 = "1lmysxfhy727zskavml3ibg5w876ir88923bm17c21s59w5lh7l8";
+      hash = "sha256-bud59LhdqCHUpTg88+gmZZKiuhCP9Zl2dU/A+Ka05P0=";
```

Comment:
You also need to update the hash below:
```
      hash = "sha256-1g7J1K9PG6B1CW0DDh4tEDrH1qewYUYgvWBpkScDL2U=";
    };

    bass_fx = {
      h = "C/bass_fx.h";
      version = "2.4.12.1";
      so = {
        i686_linux = "libs/x86/libbass_fx.so";
        x86_64-linux = "libs/x86_64/libbass_fx.so";
      };
      urlpath = "z/0/bass_fx24-linux.zip";
      hash = "sha256-Hul2ELwnaDV8TDRMDXoFisle31GATDkf3PdkR2K9QTs=";
```
You can add yourself as a maintainer if you like.

By the way, your commits are not linked to your GH user. See [here](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/troubleshooting-commits/why-are-my-commits-linked-to-the-wrong-user#commits-are-not-linked-to-any-user).

Reply:
sorry for the long wait, had a few things to do

I also updated libbass_fx and added libbassmix, should I create a new pull request and bundle all of that there?

my commits being unverified and adding myself as a maintainer are postponed until I figure out how to selfhost an email server or finally decide to temporarily buy some service; is it alright if I set my email to a non-existent one and leave my commits unverified for now?

Reply:
> is it alright if I set my email to a non-existent one and leave my commits unverified for now?

Unverified commits are fine, but we usually want a working email address in the maintainers list.